### PR TITLE
Fix asset flags.

### PR DESCRIPTION
**Pull request by winder**
Fix inconsistencies in `goal asset` command.

### CODE REVIEW

File: cmd/goal/asset.go [hunk 1]
```diff
@@ -110,7 +110,7 @@ func init() {
 	addTxnFlags(freezeAssetCmd)
 
 	infoAssetCmd.Flags().Uint64Var(&assetID, "assetid", 0, "ID of the asset to look up")
-	infoAssetCmd.Flags().StringVar(&assetUnitName, "asset", "", "Unit name of the asset to look up")
+	infoAssetCmd.Flags().StringVar(&assetUnitName, "unit", "", "Unit name of the asset to look up")
```
Comment (winder) [inline cmd/goal/asset.go:113]: I'm not sure about changing the flag name. unit is more accurate, but for I think for most people unit is also a fine name to use.

**Comment by algorandskiy** [comment]
I think a new `unit` flag needs to be added and `asset` not removed but marked as deprecated.

File: cmd/goal/asset.go (hunk 1 above)
Comment (algonautshant) [reply cmd/goal/asset.go:113]: How about following the naming convention stablished in:
```
Usage:
  goal asset create [flags]
...
      --unitname string             Name for the unit of asset
...
```

then, asset/unit would be **unitname**

Comment (algorotem) [reply cmd/goal/asset.go]: Does it actually look it up by asset (Tether USDt) or unit name (USDt)?

Comment (winder) [reply cmd/goal/asset.go]: It looks up by unit

Comment (algorotem) [reply cmd/goal/asset.go]: So seems like we should change it or follow Pavel's suggestion, no?

**Review by algonautshant — DISMISSED** [review]
I think it will be better to have the unit flag consistent within goal asset. 
Now, it is --unit here, and --unitname in asset create. 

If --unit is preferred, then changing --unitname to --unit may need it to be deprecated... I understand the complexity. 

This is just a suggestion. 

Otherwise, looks good.

**Review by algorotem — APPROVED** [review]

**Pull request merged**